### ISSUE TITLE
Bump min fee to 5000 sat/KB

### ISSUE THREAD

**Issue by jim618** [bitcoin-solutions/multibit#731]
In light of this post:
https://www.reddit.com/r/Bitcoin/comments/3oecqm/psa_to_address_mempool_flooding_bitcoin_core/

We should bump the min amount possible in the tx fee slider to 5000 sat / KB)
The code has been written so that any amount lower than that (currently minimum is 1000 sat / KB) will get lifted.

**Comment by gary-rowe** [comment]
I'm concerned that if we make this the floor then we'll have to issue a
further upgrade if it goes away or the price rises significantly.

This seems a kludge on behalf of Core.

**Comment by jim618** [comment]
Yes it is a bit of a kludge but any lower than 5000 sat/ kb and users will run into problems so I think we need to do it.
